spec/features/show_a_document_api_down_spec.rb

```diff
@@ -11,18 +11,10 @@
   end
 
   def given_there_is_a_document_with_associations
-    @document = create(:document, :with_associations_in_schema)
-
-    @document.document_type_schema.associations.each do |schema|
-      publishing_api_has_linkables([], document_type: schema.document_type)
-    end
-
-    associations = @document.document_type_schema.associations.map do |schema|
-      [schema.id, [SecureRandom.uuid]]
-    end
-
-    @document.update(associations: Hash[associations])
-    publishing_api_isnt_available
+    association_schema = attributes_for(:association, type: "multi_association")
```

Comment:
We should probably all this factory 'association_schema' to avoid confusion with an actual association.